target/linux/ath79/generic/base-files/etc/board.d/02_network

```diff
@@ -257,6 +257,10 @@ ath79_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"0@eth0" "1:lan:1" "2:lan:3" "3:lan:2"
 		;;
+	tplink,tl-wpa8630p-v2)
+		ucidef_add_switch "switch0" \
+			"0@eth0" "2:lan:3" "3:lan:2" "4:lan:1" "5:lan:4"
```

Comment:
In the commit message, you tell only about 3 ports?

Reply:
True, the fourth port is connected to the PLC. I'll clarify that in the commit message.